crates/holochain/src/conductor/conductor.rs

```diff
@@ -1396,15 +1404,22 @@ mod clone_cell_impls {
         pub async fn enable_clone_cell(
             self: Arc<Self>,
             payload: &EnableCloneCellPayload,
-        ) -> ConductorResult<InstalledCell> {
+        ) -> ConductorResult<CellInfo> {
+            let conductor = self.clone();
             let (_, enabled_cell) = self
                 .update_state_prime({
                     let app_id = payload.app_id.to_owned();
                     let clone_cell_id = payload.clone_cell_id.to_owned();
                     move |mut state| {
                         let app = state.get_app_mut(&app_id)?;
                         let clone_id = app.get_disabled_clone_id(&clone_cell_id)?;
-                        let enabled_cell = app.enable_clone_cell(&clone_id)?;
+                        let (cell_id, _) = app.enable_clone_cell(&clone_id)?.into_inner();
+                        let ribosome = conductor.get_ribosome(cell_id.dna_hash())?;
+                        let dna = ribosome.dna_file.dna();
+                        let modifiers = dna.modifiers.clone();
+                        let name = dna.name.clone();
+                        let enabled_cell =
+                            CellInfo::new_cloned(cell_id, clone_id, modifiers, name, true);
```

Comment:
@jost I wonder if the cell info should also contain the original DNA hash? Not sure, some happ devs would need to chime in. Because I imagine a common case would be to look for a clone cell based on the criteria used to create it, which would be the DNA modifiers plus the *original* DNA hash, whereas this only contains the actual modified DNA hash. @guillemcordoba @Connoropolous @matthme ?

Reply:
The cell info struct for deferred cells `StemCell` does have the DNA hash as a field. I think it doesn't hurt to include it for provisioned and cloned cells.